lib/reek/source/source_code.rb

```diff
@@ -36,14 +36,14 @@ def initialize(code, origin, parser: Parser::Ruby22)
       # @param source [File|IO|String] - the given source
       #
       # @return an instance of SourceCode
-      # :reek:DuplicateMethodCall: { max_calls: 2 }
```

Comment:
And we should keep `source.read` for readability reasons and since it has no significant negative impact (actually this whole SmellDetector should ignore duplicate calls when they happen in mutually exclusive branches, but that's another topic).